.prettierignore

```diff
@@ -7,3 +7,5 @@ api-report.md
 # Ignore the helm chart
 charts/
 e2e/playwright-report/
+
+.github/
```

Comment:
Why not format the workflow files? Does it break anything?

Reply:
prettier removes double spaces before inline comments, which my editor complains about :thinking: